### PR TITLE
[DROOLS-6338] Add documentation about invoker-test skipping

### PR DESCRIPTION
**Pull request by gitgabrio**
@danielezonca @mariofusco @jiripetrlik

### CODE REVIEW

**Comment by gitgabrio** [comment]
Jenkins retest this

**Comment by gitgabrio** [comment]
Jenkins retest this

**Comment by gitgabrio** [comment]
@mariofusco @danielezonca 
This PR changes only an MD file. Now it does not builds for some failing Kafka tests on: could this be merged anyway ? The original issue https://issues.redhat.com/browse/DROOLS-6338 was due to deployment failures

File: kie-maven-plugin/IntegrationTests.md [hunk 1]
```diff
@@ -14,6 +14,19 @@ Test compilations and reports may be found under _target/it/{module_name}_ direc
 _target/it/{module_name}/build.log_ will contain overall build output, while additionally test reports may be found under
 _target/it/{module_name}/target/failsafe-reports_ and _target/it/{module_name}/target/surefire_
 
+
+Skipping/selecting tests
+========================
+To completely disable execution of those tests, set the _invoker.skip_ parameter to false:
+
+    mvn clean install -Dinvoker.skip=true
```
Comment (danielezonca) [inline kie-maven-plugin/IntegrationTests.md:22]: What about configure `maven-invoker-plugin` also to consider `skipTests` property like it has been done in Kogito ( see https://github.com/kiegroup/kogito-runtimes/blob/master/kogito-build-parent/pom.xml#L1234-L1235 )?
I think it simplify the usage for people that are not familiar with invoker plugin (and then of course add this information to this doc)

Comment (gitgabrio) [reply kie-maven-plugin/IntegrationTests.md:22]: @danielezonca 
Thanks for suggestion. Put together that with @mariofusco observation (and general standards) I've used the "skipITs" parameter.

**Review by jiripetrlik — APPROVED** [review]

**Comment by gitgabrio** [comment]
@kiegroup/gatekeepers

Could you please merge this, squashing commits? Thanks!

**Review by mariofusco — APPROVED** [review]

**Pull request merged**